src/Mod/Mesh/Gui/Command.cpp

```diff
@@ -1129,10 +1129,10 @@ CmdMeshRemoveComponents::CmdMeshRemoveComponents()
 {
     sAppModule = "Mesh";
     sGroup = QT_TR_NOOP("Mesh");
-    sMenuText = QT_TR_NOOP("Remove components...");
-    sToolTipText = QT_TR_NOOP("Remove topologic independent components from the mesh");
+    sMenuText = QT_TR_NOOP("Remove Components…");
+    sToolTipText = QT_TR_NOOP("Removes topologic independent components from the mesh");

```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Removes topologically independent components from the mesh");
```
Or for more user friendly wording: `"Removes disconnected parts from the mesh"`
```suggestion
    sToolTipText = QT_TR_NOOP("Removes disconnected parts from the mesh");
```